File: typescript/src/scripts/run-analysis.ts
#!/usr/bin/env ts-node

import { analysisService } from '../services/analysis-service';

async function main() {
  try {
    await analysisService.runAnalysis();
  } catch (error) {
    console.error('Analysis failed:', error);
    process.exit(1);
  }
}

main();

File: python/src/scripts/run_analysis.py
#!/usr/bin/env python3

import asyncio
import sys
from src.services.analysis_service import analysis_service


async def main():
    try:
        await analysis_service.run_analysis()
    except Exception as error:
        print(f"Analysis failed: {error}")
        sys.exit(1)


if __name__ == "__main__":
    asyncio.run(main())
